src/main/java/net/snowflake/ingest/streaming/OpenChannelRequest.java

```diff
@@ -12,6 +12,7 @@ public class OpenChannelRequest {
   public enum OnErrorOption {
     CONTINUE, // CONTINUE loading the rows, and return all the errors in the response
     ABORT, // ABORT the entire batch, and throw an exception when we hit the first error
+    SKIP_BATCH, // Skip the batch after the first error, and return all the errors in the response
```

Comment:
Maybe change to `If an error in the batch is detected return a response containing all error row indexes. No data is ingested`